internal/data/common.go

```diff
@@ -424,31 +424,25 @@ func (am AttributeMap) GetAttribute(ix int) (string, AttributeValue) {
 	return (*am.orig)[ix].Key, AttributeValue{(*am.orig)[ix]}
 }
 
-// StringKeyValue stores a key and value pair.
-type StringKeyValue struct {
+// StringValue stores a key and value pair.
+type StringValue struct {
 	orig *otlpcommon.StringKeyValue
```

Comment:
If this is string value only can we make `orig` a `*string` instead?

Reply:
Would like to do that after I finalize the iterator, I may need that there. This PR does not add this new thing, and would like to avoid removing now then adding later. If ends up not being useful will change.